apps/api_web/config/config.exs

```diff
@@ -30,6 +30,26 @@ config :api_web, :rate_limiter,
   max_registered_per_interval: 100_000,
   wait_time_ms: 0
 
+config :api_web, :rate_limiter_concurrent,
+  enabled: False,
+  memcache: False,
```

Comment:
issue (blocking): boolean literals are not capitalized (https://hexdocs.pm/elixir/1.16/basic-types.html#booleans-and-nil)

Reply:
🐍 on the 🧠...